### PR TITLE
perf: cherry-pick only used lodash imports

### PR DESCRIPTION
**Pull request by TheAlexLichter**
~Use tree-shakeable version of lodash~ load lodash dependencies through sub-path

## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Refactor

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests are passing.

### CODE REVIEW

**Review by pi0 — APPROVED** [review]
Looks a good replacement. (Although i think tree-shake is not meaningful because Nuxt rollup dist externalizes dependencies.)

**Comment by manniL** [comment]
@pi0 Thank you :relaxed: 

Yeah, tree-shaking might not be that important. But fewer/smaller imports might influence the perf as well

**Review by clarkdo — COMMENTED** [review]
I suggest that we consider to merge this one after nuxt-core or since it may have some conflicts which are very hard resolve in that pr😂 @manniL @pi0

**Comment by manniL** [comment]
Sure @clarkdo! It has time :relaxed:

File: jest.config.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = {
   ],
   setupTestFrameworkScriptFile: './test/utils/setup',
   testPathIgnorePatterns: ['test/fixtures/.*/.*?/'],
-  transformIgnorePatterns: ['<rootDir>/node_modules/'],
+  transformIgnorePatterns: ['!node_modules\\/lodash-es'],
```
[the remark this answers is not in the document]
Comment (manniL) [reply jest.config.js]: Yeah, we don't want to transpile all of `node_modules` :joy: 

I'm stuck at getting Jest to work with Babel 7 (which should transpile `lodash-es` for the tests)

[the remark this answers is not in the document]
Comment (manniL) [reply jest.config.js]: Though I had a small breakthrough. let's see where this is going :)

**Comment by manniL** [comment]
Current status: 

`nuxt start` throws
```
export { default as add } from './add.js';
^^^^^^

SyntaxError: Unexpected token export
```

**Comment by manniL** [comment]
@pi0 I tried out the subpath loading ☺️

**Review by pi0 — APPROVED** [review]
🍷

File: yarn.lock [hunk 2]
```diff
@@ -6517,6 +6502,11 @@ locate-path@^3.0.0:
     p-locate "^3.0.0"
     path-exists "^3.0.0"
 
+lodash-es@^4.17.11:
```
Comment (pi0) [inline yarn.lock]: This line is remaining

File: packages/common/package.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
   "main": "dist/common.js",
   "dependencies": {
     "consola": "^1.4.4",
-    "lodash": "^4.17.11",
+    "lodash-es": "^4.17.11",
```
Comment (pi0) [inline packages/common/package.json]: Maybe because of this? :D

Comment (manniL) [reply packages/common/package.json]: Whoopa! Will revert 💃

**Pull request merged**